components/Learn/PreviousNextLink/index.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import PrevNextLink from '.';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+type Story = StoryObj<typeof PrevNextLink>;
+type Meta = MetaObj<typeof PrevNextLink>;
+
+export const Default: Story = {
```

Comment:
Could you make also stories where either previous or next is not present?

Eg: `WithoutNext` and `WithoutPrevious`